src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/BackgroundDocumentGeneratorTest.cs

```diff
@@ -37,6 +37,16 @@ public BackgroundDocumentGeneratorTest()
 
         private HostProject HostProject2 { get; }
 
+        [Fact]
+        public void TryParse()
+        {
+            if (Version.TryParse("0.18.0.0", out var result))
+            {
+                Assert.NotEqual(result.Major, -1);
+
```

Comment:
New line

Reply:
I'll stop looking at further test changes. Will do it once the PR is cleaned up.